tests/api/expected-outputs/augsburg_ar_locations.json

```diff
@@ -85,7 +85,8 @@
       "id": 1,
       "slug": "test-organization",
       "name": "Test Organization",
-      "logo": null
+      "logo": "http://localhost:8000/media/global/integreat-icon.svg",
```

Comment:
Is it secured that the server will always be at `localhost:8000` when running tests? @timoludwig

Reply:
Yes, this might only be different if somebody sets another port explicitly and runs the tests locally, but I guess in this case the person just has to deal with the failing tests... We have to assume something here, we cannot put variables in json files.
This base url is also used in all other expected api responses:
https://github.com/digitalfabrik/integreat-cms/blob/7a8fda13ca5b35a90c79077174335e01c87beb2d/tests/api/expected-outputs/augsburg_de_pages.json#L4